### ISSUE TITLE
possible enhancement:  Player height above terrain in zone properties

### ISSUE THREAD

**Issue by tarkus1971** [TheGameCreators/GameGuruRepo#570]
Zones currently are set onto the terrain, meaning if we need to check player height, for example if inside a multi storey building, maybe we could have optional 

<playerY 
>player|Y

selections in the zones properties panel.

**Comment by BerntSunde** [comment]
Yes it is a problem to be showing a picture in a room ex in a basement and then when you walk later on the floors above, the same picture pops up because you are directly above the zone, even when the zone is placed ex. 20 meters below.

The SIMPLE solution for this is to make zones 3D polygons. Well, simple for the genius programmer that is :D Then you can decide yourself how the shape of the zone is.Or you can make a zone, and then drag a handle in the middle of the zone and make it taller. Like you draw a rectangle in Sketchup and then drag it to become a 3d box.

**Comment by tarkus1971** [comment]
yes, polygon or box would be very very useful and would allow for much more detailed interaction with things.
